bookShop/src/app/app.component.ts

```diff
@@ -15,16 +22,11 @@ export class AppComponent implements OnInit, AfterViewInit {
 
   basketData: IBook[] = [];
 
-  bookData: IBook[] = [];
-
-  constructor(private cartService: CartService, private bookService: BooksService) {
-    this.cartService.clickEvent.subscribe(cnt => this.basketData = cnt);
-    this.cartService.removeEvent.subscribe(cnt => this.bookData = cnt);
+  constructor(private cartService: CartService) {
+    this.cartService.clickEvent.subscribe((cnt) => (this.basketData = cnt));
```

Comment:
Change the name of `cnt` to smth. more understandable.

Reply:
Corrected